src/cpp/session/modules/ModuleTools.R

```diff
@@ -18,6 +18,12 @@
    .Call("rs_enqueClientEvent", type, data)
 })
 
+.rs.addFunction("invokeRpc", function(method, ...) 
+{
+   # callback to session to invoke RPC
+   .Call("rs_invokeRpc", method, c(...))
```

Comment:
`list(...)` is preferred here since it'll be type-stable. E.g.

```
> test <- function(...) { c(...) }
> test(1, "2", 3)
[1] "1" "2" "3"
```

Reply:
Yes, I think I had that at one point, but the RPC marshaling turned the list into an array. I took another look and discovered that we probably just need to signal that the single elements are scalars (this is not 100% foolproof but has few exceptions). I've just made that change. Thanks for the pointer!